microsoft-edge/extensions-chromium/store-policies/ada-addendum.md

```diff
@@ -14,7 +14,7 @@ ms.date: 02/17/2021
 <!-- ====================================================================== -->
 ## Exhibit G: Terms and Conditions for Microsoft Edge Extensions
 
-These Terms and Conditions for Microsoft Edge Extensions (the "Extensions Addendum") is a supplement to the terms and conditions of the [App Developer Agreement](https://go.microsoft.com/fwlink/p/?LinkID=221922).  This Addendum applies if you have submitted or are making available Microsoft Edge Extensions (each, an "Extension") through Microsoft Edge Add-ons.  Except as expressly modified by this Extensions Addendum, all of the terms and conditions in the App Developer Agreement, which is incorporated by reference herein, apply to the offering and distribution of your Microsoft Edge Extensions.  Except where expressly modified by this Addendum, the terms and conditions that apply to an "Application" or "App" under the App Developer Agreement will apply to an Extension as defined in this Addendum and all references to "Store" or "Microsoft Store" under the App Developer Agreement will apply to Microsoft Edge Add-ons as defined in this Addendum.
+These Terms and Conditions for Microsoft Edge Extensions (the "Extensions Addendum") is a supplement to the terms and conditions of the [Microsoft Store App Developer Agreement](https://go.microsoft.com/fwlink/p/?LinkID=221922).  This Addendum applies if you have submitted or are making available Microsoft Edge Extensions (each, an "Extension") through Microsoft Edge Add-ons.  Except as expressly modified by this Extensions Addendum, all of the terms and conditions in the App Developer Agreement, which is incorporated by reference herein, apply to the offering and distribution of your Microsoft Edge Extensions.  Except where expressly modified by this Addendum, the terms and conditions that apply to an "Application" or "App" under the App Developer Agreement will apply to an Extension as defined in this Addendum and all references to "Store" or "Microsoft Store" under the App Developer Agreement will apply to Microsoft Edge Add-ons as defined in this Addendum.
 
 1.  DEFINITIONS.  Capitalized terms not defined herein will have the meanings provided in the App Developer Agreement.
 
```

Comment:
line 21: normalize link that shows a fwlink as link text.  link text should match destination title per corp style.  
link text should not be a url.  nonstandard and confusing/disorienting.  hard to maintain.


rendered article section:
https://review.learn.microsoft.com/microsoft-edge/extensions-chromium/store-policies/ada-addendum?branch=pr-en-us-3255#exhibit-g-terms-and-conditions-for-microsoft-edge-extensions

[Before/Live](https://learn.microsoft.com/microsoft-edge/extensions-chromium/store-policies/ada-addendum#exhibit-g-terms-and-conditions-for-microsoft-edge-extensions)

Reply:
result:
![image](https://github.com/user-attachments/assets/adc46582-05c8-41ff-800d-aa6c53f99713)